_includes/v21.2/prod-deployment/healthy-read-amplification.md

```diff
@@ -0,0 +1 @@
+**Expected values for a healthy cluster**: Read amplification factor should be in the single digits. A value higher than 50 strongly suggests that the LSM tree has an unhealthy shape.
```

Comment:
"Higher than 50" was gleaned from this [postmortem](https://cockroachlabs.atlassian.net/wiki/spaces/TS/pages/1548484769/2021-03-06+Postmortem+on+DevSisters+7788)

Reply:
See: https://github.com/cockroachlabs/cockroachdb-runbook-template/blob/main/monitoring-alerts/alert-lsm-health.md

Add nuance on duration of high read amp value